schemas/IDLs/org.opencb.biodata.models/1.3.0-SNAPSHOT/avro/variantLegacy.avdl

```diff
@@ -0,0 +1,55 @@
+
+@namespace("org.opencb.biodata.models.variant.avro.legacy")
+
+protocol VariantsLegacy {
+
+    record VcfHeader {
+        string fileFormat;
+    //map<union { string, array<map<string> }>> meta; ??
+        map<array<union { string, map<string> }>> meta;
+    }
+
+    record VariantGlobalStats {
+
+        int numRecords;
+
+        int samplesCount;
+
+        int passCount;
+
+        int transitionsCount;
+
+        int transversionsCount;
+
+        double meanQuality;
+
+        map<int> variantTypeCounts;
+
+        map<int> chromosomeCounts;
+
+        map<int> consequenceTypesCount;
+
+    }
+
+    record VariantSource {
+        string fileId;
+
+        string studyId;
+
+        union { null, string } fileName = null;
+
+        union { null, string } studyName = null;
+
+        array<string> samples = [];
+
+        string aggregation;
+
+        union {null, VariantGlobalStats} stats;
+
+        map<union {string, VcfHeader}> metadata;
+
+        union {null, VcfHeader} header;
+
+        null pedigree;
+    }
+}
```

Comment:
newline please

Reply:
This is an external model, we commit this only to enable maven independent builds. Better not to change unless there is a strong reason.

Reply:
it's not a change to the model, just a change to our local version of the file